media-plugins/vdr-imonlcd/vdr-imonlcd-1.0.2.ebuild

```diff
@@ -1,29 +1,33 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
We usually revbump while doing EAPI bump, but I see this has ~ KEYWORDS and a bump from 6 -> 7 is rather minimal. 

FYI you'll get a bug report soon ;)
```
 * Messages for package media-plugins/vdr-imonlcd-1.0.2:

 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib64/vdr/plugins/libvdr-imonlcd.so.2.4.6
>>> Auto-cleaning packages...
```

Reply:
Hrmpf.... forgot to add  the usual
`
QA_FLAGS_IGNORED="
    usr/lib/vdr/plugins/libvdr-.*
    usr/lib64/vdr/plugins/libvdr-.*"
`
like on the other vdr plugins.